homeassistant/components/datadog/strings.json

```diff
@@ -0,0 +1,55 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Enter your Datadog agent's address and port.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "prefix": "Prefix",
+          "rate": "Rate"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of the Datadog agent.",
+          "port": "Port the Datadog agent is listening on",
+          "prefix": "Metric prefix to use",
+          "rate": "The sample rate of UDP packets sent to Datadog."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
```

Comment:
We need an abort `cannot_connect` as well (as you have that as an abort reason when import fails on connection issues.)